components/collapsible/collapsible.vue

```diff
@@ -11,9 +11,6 @@
         'd-dt-collapsibe__anchor',
```

Comment:
This should be d-dt-collapsible__anchor ? 🤔

Reply:
yep that is definitely a typo